group_vars/all/vars.yml

```diff
@@ -1,4 +1,5 @@
 ---
 admin_ranges: "129.125.249.0/24,172.23.40.1/24"
-ssh_host_signer_hostnames: "{{ ansible_fqdn }},{{ ansible_hostname }},airlock+{{ ansible_hostname }}"
+# ssh_host_signer_hostnames: "{{ ansible_fqdn }},{{ ansible_hostname }},airlock+{{ ansible_hostname }}"
```

Comment:
Why are you committing code in comment?

Reply:
Haven't been able to test this (yet)... If it turns out to work, then the commented line can go.